### ISSUE TITLE
Suggestions for improvements to the external tool installation process

### ISSUE THREAD

**Issue by smorimoto** [actions/runner#1011]
- Add support for the 32 bits platform (For testing on a 32 bits platform using the container feature)
- Upgrade Node.js to latest LTS release

For now, we do that in file https://github.com/actions/runner/blob/main/src/Misc/externals.sh, but the URLs used for installation are inconsistently.
Nevertheless, using the official distfiles from the Node.js team is not ideal because there is no one for 32 bits.
It's not a bad idea to create a repository for building packages for the runner and build things like Node.js there, what do you think?

**Comment by hross** [comment]
Yes I think these are good ideas but right now they are not highest priority for us. I'll label this and we will evaluate time frames on if/when we would upgrade to latest LTS.

I would be interested to see more feedback on 32 bit platform support and how many customers need this feature. Since it requires more support from us and may be a smaller subset of use cases, this feels like something we would be less likely to support.

**Comment by smorimoto** [comment]
For example, in this case https://github.com/actions/checkout/issues/334, it becomes a problem. Also the error message is very unfriendly.

**Comment by provgregoryabdo** [comment]
[https://github.com/actions/checkout/issues/334](issues/334)  Seems to makes using a "container" in the workflow virtually useless since you can't actually run any of the GitHub "actions" in it.

**Comment by smorimoto** [comment]
I can also connect representatives of the Node.js team with you as needed and ask them to release dist files for the 32 bits platform. In any case, what we need now is a response from you all.

**Comment by Flole998** [comment]
> I can also connect representatives of the Node.js team with you as needed and ask them to release dist files for the 32 bits platform. In any case, what we need now is a response from you all.

I also stumbled across this after trying to run some actions on 32-Bit containers which still fail with an error message that's super misleading. So for me it would be great if Node.js would release dist files for 32 bit (and if this is then picked up here so it can be used in actions).

**Comment by smorimoto** [comment]
The Node team's unofficial build currently has a build for 32-bit systems. I'm going to open a PR to correct the script to use it, do you have any objections?
